python/interpret_community/tabular_explainer.py

```diff
@@ -189,13 +189,15 @@ def __init__(self, model, initialization_examples, explain_subset=None, features
                         allow_all_transformations=allow_all_transformations,
                         **kwargs)
                 self._method = self.explainer._method
-                self._logger.info('Initialized valid explainer {} with args {}'.format(self.explainer, kwargs))
+                exp_init = 'Initialized valid explainer {} with args {}'.format(self.explainer, kwargs)
+                self._logger.info(exp_init)
                 is_valid = True
                 break
             except Exception as ex:
                 last_exception = ex
-                self._logger.info('Failed to initialize explainer {} due to error: {}'
-                                  .format(uninitialized_explainer, ex))
+                exp_init_failed = ('Failed to initialize explainer {} due to error: {}'
```

Comment:
Can we rename to exp_init_failed_message?

Reply:
done